### PR TITLE
moltenvk: add 1.1.0 + few improvements

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **moltenvk/1.1.1**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

moltenvk/1.1.0 added -even if 1.1.1 is already in CCI- for the reasons explained in https://github.com/conan-io/conan-center-index/pull/4760#issue-583113498

### CODE REVIEW

**Comment by SpaceIm** [comment]
I don't know if I should turn `MoltenVKShaderConverter` into a static lib instead of object target, this lib is created in upstream XCode files.
On the other hand this lib is not packaged in Vulkan SDK of LunarG, only `MoltenVK.a` (and dylib) which is a big archive of all dependencies.

**Review by uilianries — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

File: recipes/moltenvk/config.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 versions:
   "1.1.1":
     folder: all
+  "1.1.0":
```
Comment (prince-chrismc) [inline recipes/moltenvk/config.yml:4]: why are you adding an older version?

**Review by prince-chrismc — APPROVED** [review]

File: recipes/moltenvk/config.yml (hunk 1 above)
Comment (SpaceIm) [reply recipes/moltenvk/config.yml:4]: https://github.com/conan-io/conan-center-index/pull/4762#issue-583194666

Comment (prince-chrismc) [reply recipes/moltenvk/config.yml:4]: Thanks I skipped over that.

**Pull request merged**